crates/bevy_core_pipeline/src/tonemapping/node.rs

```diff
@@ -48,17 +63,21 @@ impl Node for TonemappingNode {
         let view_entity = graph.get_input_entity(Self::IN_VIEW)?;
         let pipeline_cache = world.resource::<PipelineCache>();
         let tonemapping_pipeline = world.resource::<TonemappingPipeline>();
+        let gpu_images = world.get_resource::<RenderAssets<Image>>().unwrap();
+        let view_uniforms_resource = world.resource::<ViewUniforms>();
+        let view_uniforms = view_uniforms_resource.uniforms.binding().unwrap();
 
-        let (target, tonemapping) = match self.query.get_manual(world, view_entity) {
-            Ok(result) => result,
-            Err(_) => return Ok(()),
-        };
+        let (view_uniform_offset, target, view_tonemapping_pipeline, tonemapping) =
+            match self.query.get_manual(world, view_entity) {
+                Ok(result) => result,
+                Err(_) => return Ok(()),
```

Comment:
This is an unusual pattern and looks like an error to a casual reader. A comment here would be appreciated.

Reply:
Is the unusual thing related to something I added? The general pattern with `self.query.get_manual(world, view_entity)` is only something I've added to. This general pattern is common in rendering nodes and appears at least 4 other times.

Reply:
I think the `return` statement diverting control flow and returning `Ok(())` is easy to miss? When first reading this I was like “why not `.unwrap_or()?” until I saw the `return`.

Maybe a `let ... else` statement with the `Ok()` on the lefthand side would make the `return` more apparent as its own statement?

Reply:
This was pre-existing: you only added an indentation change. Definitely made this function harder to review though.